jf_agent/main.py

```diff
@@ -499,6 +505,11 @@ def upload_file(filename, path_to_obj, signed_url, local=False):
         t.join()
 
     if any(thread_exceptions):
+        # Run through exceptions and inject them into the agent log
+        for exception in thread_exceptions:
+            agent_logging.log_and_print_error_or_warning(
+                logger, logging.ERROR, error_code=1000, msg_args=[exception], exc_info=True
```

Comment:
I think you mean to use the code `0000`?

Reply:
Oh, right. Good catch